--- a/docs/config.toml
+++ b/docs/config.toml
@@ -1,4 +1,4 @@
-baseURL = "/v0.11"
+baseURL = "/"
 title = "HAProxy Ingress"
 
 enableRobotsTXT = true
@@ -81,15 +81,15 @@ privacy_policy = ""
 
 # Menu title if your navbar has a versions selector to access old versions of your site.
 # This menu appears only if you have at least one [params.versions] set.
-version_menu = "v0.11"
+version_menu = "v0.11 (latest)"
 
 [[params.versions]]
-version = "Latest"
-url = "https://haproxy-ingress.github.io/docs"
+version = "v0.12 (snapshot)"
+url = "https://haproxy-ingress.github.io/v0.12/docs"
 
 [[params.versions]]
-version = "v0.11"
-url = "https://haproxy-ingress.github.io/v0.11/docs"
+version = "v0.11 (latest)"
+url = "https://haproxy-ingress.github.io/docs"
 
 [[params.versions]]
 version = "v0.10"
