--- a/docs/config.toml
+++ b/docs/config.toml
@@ -81,15 +81,15 @@ privacy_policy = ""
 
 # Menu title if your navbar has a versions selector to access old versions of your site.
 # This menu appears only if you have at least one [params.versions] set.
-version_menu = "v0.14"
+version_menu = "v0.14 (latest)"
 
 [[params.versions]]
-version = "Latest"
-url = "/docs"
+version = "v0.15 (snapshot)"
+url = "/v0.15/docs"
 
 [[params.versions]]
-version = "v0.14"
-url = "/v0.14/docs"
+version = "v0.14 (latest)"
+url = "/docs"
 
 [[params.versions]]
 version = "v0.13"
