--- a/docs/config.toml
+++ b/docs/config.toml
@@ -1,4 +1,4 @@
-baseURL = "/v0.14"
+baseURL = "/"
 title = "HAProxy Ingress"
 
 enableRobotsTXT = true
@@ -81,16 +81,12 @@ privacy_policy = ""
 
 # Menu title if your navbar has a versions selector to access old versions of your site.
 # This menu appears only if you have at least one [params.versions] set.
-version_menu = "v0.14"
+version_menu = "v0.14 (latest)"
 
 [[params.versions]]
-version = "Latest"
+version = "v0.14 (latest)"
 url = "https://haproxy-ingress.github.io/docs"
 
-[[params.versions]]
-version = "v0.14"
-url = "https://haproxy-ingress.github.io/v0.14/docs"
-
 [[params.versions]]
 version = "v0.13"
 url = "https://haproxy-ingress.github.io/v0.13/docs"
